apps/SageAccountWeb/src/LoginPage.tsx

```diff
@@ -1,28 +1,28 @@
 import React from 'react'
 import { Box, Typography } from '@mui/material'
-import StandaloneLoginForm from 'synapse-react-client/dist/containers/auth/StandaloneLoginForm'
-import { SourceAppDescription, SourceAppLogo } from './components/SourceApp'
-import {
-  StyledInnerContainer,
-  StyledOuterContainer,
-} from './components/StyledComponents'
 import {
   preparePostSSORedirect,
   redirectAfterSSO,
-} from 'synapse-react-client/dist/utils/AppUtils'
+  StandaloneLoginForm,
+  SynapseConstants,
+  SystemUseNotification,
+  useApplicationSessionContext,
+} from 'synapse-react-client'
+import { SourceAppDescription, SourceAppLogo } from './components/SourceApp.js'
+import {
+  StyledInnerContainer,
+  StyledOuterContainer,
+} from './components/StyledComponents.js'
 import { useHistory } from 'react-router-dom'
-import { useApplicationSessionContext } from 'synapse-react-client/dist/utils/apputils/session/ApplicationSessionContext'
-import { backButtonSx } from './components/BackButton'
-import { LOGIN_BACK_BUTTON_CLASS_NAME } from 'synapse-react-client/dist/utils/SynapseConstants'
-import SystemUseNotification from 'synapse-react-client/dist/containers/SystemUseNotification'
+import { backButtonSx } from './components/BackButton.js'
 
 export type LoginPageProps = {
   returnToUrl?: string
 }
 
 function LoginPage(props: LoginPageProps) {
   const { returnToUrl } = props
-  const { refreshSession, twoFactorAuthErrorResponse } =
+  const { refreshSession, twoFactorAuthSSOErrorResponse } =
```

Comment:
Was using the wrong name for the object. I have a feeling this will fix https://sagebionetworks.jira.com/browse/PORTALS-2643 but need to confirm